conda-recipes/llvmlite/meta.yaml

```diff
@@ -23,19 +23,19 @@ requirements:
     # build.sh deals with it!
     - {{ compiler('c') }}    # [not (osx or armv6l or armv7l or win)]
     - {{ compiler('cxx') }}  # [not (osx or armv6l or armv7l or win)]
-    - vs2017_{{ target_platform  }}    # [win]
+    - vs2015_{{ target_platform  }}    # [win]
```

Comment:
I had problems with generating cmakes files on Windows. Things seem to have changed on the LLVM side. I can try going to 2017 again, but I think 2019 isn't supported yet.

Reply:
> but I think 2019 isn't supported yet.

It's the other way around. As of LLVM 14 (more precisely: https://reviews.llvm.org/D114639), VS2019 is the minimally required version

Reply:
VS2019 doesn't seem to be available in conda yet. I'm pretty much at the "🤷  this builds" point with Windows.

Reply:
It is, but you get it through the regular `{{ compiler('cxx') }}`. Conda actually switched to VS2019 by default some time ago, but I haven't checked out how you hook into that infrastructure. Otherwise those are specified via `conda_build_config.yaml`

Reply:
FWIW, [here](https://conda-forge.org/docs/maintainer/knowledge_base.html#using-vs2019) are the instructions for using vs2019 within conda-forge (from before the switch to the new default, but describe the necessary todos).